src/app/operations/operations.module.ts

```diff
@@ -0,0 +1,22 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { OperationsContainerComponent } from './operations-container.component';
+import { OperationContainerComponent } from './operation-container.component';
+import { OperationDetailsComponent } from './operation-details.component';
+import { OperationSamplesComponent } from './operation-samples.component';
+import { OperationResourceComponent } from './operation-resource.component';
+import { OperationPathComponent } from './operation-path.component';
+import { OperationParametersComponent } from './operation-parameters.component';
+import { OperationSampleRequestComponent } from './operation-sample-request.component';
+import { OperationParameterComponent } from './operation-parameter.component';
+
+@NgModule({
+  imports: [
+    CommonModule
+  ],
+  exports: [
+    OperationsContainerComponent
+  ],
+  declarations: [OperationContainerComponent, OperationDetailsComponent, OperationSamplesComponent, OperationsContainerComponent, OperationResourceComponent, OperationPathComponent, OperationParametersComponent, OperationSampleRequestComponent, OperationParameterComponent]
```

Comment:
I'm definitely +1 on limiting line length (although I definitely have some long ones in here that I didn't address), but I think adding lint rule will make it so we have to be manually adjusting our module files to be lint free (since ng-cli modifies them when generators are used), or ignoring the rule on those files. I'd rather just handle line length on a case-by-case basis.

I also have concerns about how this would play out with some of the templates, as our `<pre>` tags can't be broken up arbitrarily.